packages/neuron-ui/src/components/NervosDAO/index.tsx

```diff
@@ -263,28 +272,32 @@ const NervosDAO = () => {
           )}
         </div>
       }
-      notice={pageNotice}
     >
       <div className={styles.header}>
         <div className={styles.daoOverview}>
           <div className={clsx(styles.field, styles.free)}>
             <div className={styles.name}>{t(`nervos-dao.free`)}</div>
             <div className={styles.value}>
-              <CopyZone
-                content={shannonToCKBFormatter(`${free}`, false, '')}
-                name={t('nervos-dao.copy-balance')}
-                className={styles.balance}
-              >
-                <span className={styles.number}>{isPrivacyMode ? HIDE_BALANCE : shannonToCKBFormatter(`${free}`)}</span>{' '}
-                CKB
-              </CopyZone>
+              {isPrivacyMode ? (
+                <>
+                  <span className={styles.number}>{HIDE_BALANCE}</span> CKB
```

Comment:
I find there were many `HIDE_BALANCE` in Neuron, maybe we can extract them as a component in the future

```tsx
interface Balance {
  value?: Numberish;
  hidden?: boolean;
  copyable?: boolean;
  formatter?: string; // '0,0.[0000]'
  suffix?: ReactElement; // CKB or something else
}
```

Reply:
A good suggestion, it can refactor later.